integration/spark/spark3/src/test/java/io/openlineage/spark3/agent/lifecycle/plan/DataSourceV2RelationDatasetBuilderTest.java

```diff
@@ -106,50 +109,4 @@ private static Stream<Arguments> provideBuilders() {
             factory,
             openLineage));
   }
-
-  @ParameterizedTest
-  @MethodSource("provideBuildersWithSparkListeners")
-  public void testIsDefinedStSparkListenerEvent(
```

Comment:
Minor: It's not obvious why `testIsDefinedStSparkListenerEvent()` was removed.

Reply:
It's always true now. I don't see a point to a test